src/components/Sing/CharacterMenuButton.vue

```diff
@@ -188,9 +188,11 @@ export default defineComponent({
       const characterInfo = userOrderedCharacterInfos.value?.find(
         (info) => info.metas.speakerUuid === speakerUuid
       );
-      const defaultStyleId = store.state.defaultStyleIds.find(
-        (x) => x.speakerUuid === speakerUuid
-      )?.defaultStyleId;
+
+      // FIXME: ソングエディタ向けのデフォルトスタイルを選択できるようにする
+      // ここで取得されるcharacterInfoには、ソングエディタ向けのスタイルのみ含まれるので、
+      // その中の最初のスタイルをソングエディタにおけるデフォルトスタイルとする
+      const defaultStyleId = characterInfo?.metas.styles[0].styleId;
```

Comment:
ソングのデフォルトスタイル、ぶっちゃけなくても良い気がちょっとしてます。
トークみたく何度もキャラ変更しないよなーと。 
「初期値の優先順位をソング→ノーマル→その他にする」みたいなのでも良いかも…？
いやハミングを優先したい人もいそう･･･。

ということで、FIXMEコメントに幅をもたせとくのどうでしょう！
（`仮の`とし、FIXMEをTODOにして下に持っていき、「どうするか考える」にしてます）
```suggestion
      // ここで取得されるcharacterInfoには、ソングエディタ向けのスタイルのみ含まれるので、
      // その中の最初のスタイルをソングエディタにおける仮のデフォルトスタイルとする
      // TODO: ソングエディタ向けのデフォルトスタイルをどうするか考える
      const defaultStyleId = characterInfo?.metas.styles[0].styleId;
```